### PR TITLE
fix:text-only-card-variants

### PR DESCRIPTION
**Pull request by anshtripathi01**
**_Please Review_**
fix-card

### CODE REVIEW

File: Components/Cards/cards.css [hunk 1]
```diff
@@ -101,8 +101,19 @@ margin: 1rem;
...
 .text-container{
     width: 15rem;
-    height: 14rem;
-    
+    height: 15rem;
+}
+.text-only-card{
+    box-shadow: 0px 0px 8px 0px rgba(0,0,0, 0.1);
+    width: 240px;
+    height: 200px;
```
Comment (LordUttam) [inline Components/Cards/cards.css:109]: Is there a reason why you are using px here? I think you can use rems

File: Components/Cards/cards.css [hunk 2]
```diff
@@ -101,8 +101,19 @@ margin: 1rem;
 }
 .text-container{
     width: 15rem;
-    height: 14rem;
-    
+    height: 15rem;
+}
+.text-only-card{
+    box-shadow: 0px 0px 8px 0px rgba(0,0,0, 0.1);
```
Comment (LordUttam) [inline Components/Cards/cards.css:107]: Suggestion to make variable for this color like --primary-shadow-color.

File: Components/Cards/cards.css [hunk 3]
```diff
@@ -10,7 +10,7 @@ body{
     flex-direction: column;
     justify-content: center;
     align-items: center;
-    width: 55rem;
+    width: 56rem;
     height: 55rem;
     box-shadow: 0px 1px 4px 0px rgba(0, 0, 0, 0.3);
```
Comment (LordUttam) [inline Components/Cards/cards.css:15]: Same here as well.

**Review by LordUttam — APPROVED** [review]
I don't see any mistakes but made some suggestions.

File: Components/Cards/cards.css [hunk 4]
```diff
@@ -10,7 +10,7 @@ body{
     flex-direction: column;
     justify-content: center;
     align-items: center;
-    width: 55rem;
```
Comment (AtulHande03a) [inline Components/Cards/cards.css:13]: you can try using width and height in % .

**Review by AtulHande03a — APPROVED** [review]
rest looks fine !!

File: Components/Cards/cards.css (hunk 2 above)
Comment (anshtripathi01) [reply Components/Cards/cards.css:107]: ok. done

File: Components/Cards/cards.css (hunk 4 above)
Comment (anshtripathi01) [reply Components/Cards/cards.css:13]: ok

**Pull request merged**